src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -197,6 +198,7 @@ stop_metadata_sync_to_node(PG_FUNCTION_ARGS)
 
 	text *nodeName = PG_GETARG_TEXT_P(0);
 	int32 nodePort = PG_GETARG_INT32(1);
+	bool clearMetadata = PG_GETARG_BOOL(2);
```

Comment:
It seems that we do not reverse what `LocalGroupIdUpdateCommand()` does in `start_metaata_sync_to_node()`.

I think we should be resetting that as well.

Are there any other things that `start_metadata_Sync` does and `stop_metadata_sync_to_node ` does not reverse? I think ideally we should reverse everything.